web/src/api/auth/login-callback.ts

```diff
@@ -124,6 +142,12 @@ export const auth0Login = withApiAuthRequired(
       },
     });
 
-    return res.redirect(307, urls.app());
+    return res.redirect(
+      307,
+      urls.app(
```

Comment:
let's add a comment that we're redirecting here because the user already has an account and you can't join another team (yet)